src/qt/locale/bitcoin_pl.ts

```diff
@@ -3735,6 +3756,44 @@ Wartości poniżej 0.546*minimalna wartość przekazu. są traktowane jako &quot
         <location line="+1"/>
         <source>When creating transactions, ignore inputs with value less than this (default: 0.01)</source>
         <translation>Przy tworzeniu transakcji, ignoruj wpisy większe niż to (domyślnie: 0.01)</translation>
+        <location line="+1"/>
```

Comment:
Are you sure this is correct? Can one message have two locations?

Reply:
This is incorrect. There should be <message> in front of it. I added it to few lines. I could miss this one.

Reply:
The second location should go.

If you translate the file in Qt Linguist it will be a bit easier, if you aren't already.

Reply:
I did translate it in Qt, though I don't know of a way to merge two files in it.